.github/workflows/live-skills.yml

```
@@ -0,0 +1,56 @@
+          title="Live skills integration schedule is failing"
+          run_url="$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID"
+          existing=$(gh issue list --repo "$GITHUB_REPOSITORY" --state open \
+            --search "\"$title\" in:title" --json number --jq '.[0].number // empty')
+          if [ -n "$existing" ]; then
+            gh issue comment "$existing" --repo "$GITHUB_REPOSITORY" \
+              --body "Still failing: $run_url"
+          else
+            gh issue create --repo "$GITHUB_REPOSITORY" --title "$title" \
+              --body "The scheduled live skills integration run failed: $run_url
+
+The job runs \`make live-skills-test\` against the real \`npx skills\` CLI; a failure here usually means upstream drift in the skills CLI rather than a regression in this repo. Reproduce locally with \`make live-skills-test\`."
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)